node/nodeRunner.go

```diff
@@ -491,7 +491,7 @@ func (nr *nodeRunner) createApiFacade(
 
 func (nr *nodeRunner) createHttpServer() (shared.UpgradeableHttpServerHandler, error) {
 	httpServerArgs := gin.ArgsNewWebServer{
-		Facade:          disabled.NewDisabledNodeFacade(nr.configs.FlagsConfig.RestApiInterface),
+		Facade:          initial.NewInitialNodeFacade(nr.configs.FlagsConfig.RestApiInterface, nr.configs.FlagsConfig.EnablePprof),
```

Comment:
could have provided the whole flags config struct.

Reply:
no need for this now as it only uses 2 fields of the huge list of flags. maybe if we need another or a few more flags, we can parse the whole struct

Reply:
ok